test/integration/suites/node-attestation/00-setup

```diff
@@ -5,6 +5,8 @@ echo ${ROOTDIR}
 # Move test x509pop certificate and key
 mv conf/agent.key.pem conf/agent/test.key.pem
 mv conf/agent.crt.pem conf/agent/test.crt.pem
+# add read access to prevent error when reading with user 1001
+chmod +x conf/agent/test.key.pem
```

Comment:
Isn't this adding the executable bit? How does this fix read permissions?

Reply:
🤦‍♂️ thanks for noting this @azdagron!